### PR TITLE
Displayed the Download Size for Each dataset

### LINKED ISSUE #120: [Documentation] Display the download size for each dataset.
Currently, just by looking at the [list of datasets available in TFDS](https://github.com/tensorflow/datasets/blob/master/docs/datasets.md#all-datasets), there is no way to know the size of each dataset prior to downloading. Users may be operating under constrained disk space, and should be informed of the size of the dataset before requesting.

This feature enhancement would **detail the download size of each dataset** on the markdown file referenced above.

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes #120 

- Display the download size for each dataset.
- I have converted the size to MB for human-readable Formatting.
- Added an extra column and created a Markdown table for it.

@Conchylicultor @dynamicwebpaige Please review and let me know if I have to do any changes.

Thanks

### CODE REVIEW

**Comment by ParthS007** [comment]
@rsepassi Please review and let me know if I have to do any changes.

Thanks

File: docs/datasets.md [hunk 1]
```diff
@@ -29,47 +29,69 @@ np_datasets = tfds.as_numpy(datasets)
 ## All Datasets
 
 * [`audio`](#audio)
```
Comment (cyfra) [inline docs/datasets.md]: This file is auto-generated.

You should change the generator instead: scripts/document_datasets.py

Comment (ParthS007) [reply docs/datasets.md]: @cyfra Should I make a new PR for that and close this one or make changes here only?

Comment (Conchylicultor) [reply docs/datasets.md]: Thanks for your contribution. I think https://github.com/tensorflow/datasets/pull/145 is already accomplishing this so we could probably close this one.

Comment (ParthS007) [reply docs/datasets.md]: okay :+1:

**Pull request closed without merging**